tests/unit/routes/authors-test.js

```diff
@@ -1,18 +1,17 @@
-import { module, test } from 'qunit';
-import { setupTest } from 'ember-qunit';
-import EmberObject from '@ember/object';
+import {module, test} from 'qunit';
+import {setupTest} from 'ember-qunit';
 import sinon from 'sinon';
 
-const { spy, stub } = sinon;
+const {stub} = sinon;
+
 
 module('Unit | Route | authors', hooks => {
   setupTest(hooks);
 
-
   hooks.beforeEach(function() {
```

Comment:
Let's get rid of `beforeEach` since we only have one test now.

Reply:
Thanks. Done.